### PR TITLE
Version 1.9

### PR DESCRIPTION
**Pull request by antoinekociuba-artifakt**

### CODE REVIEW

**Review by amatheossian1 — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @Sylius/core-team
```
Comment (bcadiot) [inline .github/CODEOWNERS]: Please remove this file
I don't want Sylius to have access to our github repo

File: .env.prod [hunk 2]
```diff
@@ -0,0 +1,2 @@
+APP_DEBUG=0
+APP_SECRET='s$cretf0rt3st'
```
Comment (bcadiot) [inline .env.prod]: How this secret is changed for prod env?

File: artifakt/setup.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+#!/bin/sh
```
Comment (bcadiot) [inline artifakt/setup.sh:1]: These files are only compatible with stack version 4.X not the upcoming 5.X

File: docker-compose.prod.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+services:
+  php:
+    image: quay.io/sylius/php:latest
+    depends_on:
+      - mysql
+    environment:
+      - APP_ENV=prod
+      - APP_DEBUG=0
+      - APP_SECRET=EDITME
```
Comment (bcadiot) [inline docker-compose.prod.yml:11]: Is it the secret we use in production?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,59 @@
+{
+  "dependencies": {
+    "babel-polyfill": "^6.26.0",
+    "chart.js": "^2.9.3",
+    "jquery": "^3.5.0",
+    "jquery.dirtyforms": "^2.0.0",
+    "lightbox2": "^2.9.0",
+    "semantic-ui-css": "^2.2.0",
+    "slick-carousel": "^1.8.1"
+  },
```
Comment (bcadiot) [inline package.json:10]: I'm not particularly confortable to manage these extensions, what happened if a security issue is raised on one of these dependencies ?
Another point for using submodules, or at least using packages produced by editors

File: package.json [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  "scripts": {
+    "build": "gulp build",
+    "gulp": "gulp build",
+    "lint": "yarn lint:js",
+    "lint:js": "eslint gulpfile.babel.js",
+    "watch": "gulp watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Sylius/Sylius.git"
```
Comment (bcadiot) [inline package.json:55]: That's not us

Comment (bcadiot) [reply package.json:55]: One of the problem of duplicating code, who is the owner of our code ?

**Review by bcadiot — CHANGES_REQUESTED** [review]
- Please remove the CODEOWNERS file
- Is this assumed that this will not work with Artifakt stack version 5.0 ?
- It would be better to use submodules to link this repo rather than copy pasting everything
- or using packagist : https://packagist.org/packages/sylius/sylius

**Comment by djalal** [comment]
.github folder should be removed for now, it triggers a matrix of checks, many of which we don't support on Artifakt PaaS. please disable github actions until we get the proper CI pipeline.

**Pull request closed without merging**